backend/accounts_test.go

```diff
@@ -179,3 +182,86 @@ func TestNextAccountNumber(t *testing.T) {
 	_, err = nextAccountNumber(coinpkg.CodeTBTC, ks(fingerprint2, true), accountsConfig)
 	require.Equal(t, ErrAccountLimitReached, errp.Cause(err))
 }
+
+func newBackend(t *testing.T, testing, regtest bool) *Backend {
+	t.Helper()
+	b, err := NewBackend(
+		arguments.NewArguments(
+			test.TstTempDir("appfolder"),
+			testing, regtest,
+			false, false,
+			&types.GapLimits{Receive: 20, Change: 6}),
+		nil,
+	)
+	require.NoError(t, err)
+	return b
+}
+
+func TestSupportedCoins(t *testing.T) {
+	// All coins supported, mainnet.
+	t.Run("", func(t *testing.T) {
```

Comment:
The comments should actually be sub test names. It's what it's made for when watching tests fail - tells you which one:

```go
t.Run("all coins supported, mainnet", ...)
t.Run("all coins supported, testnet", ...)
t.Run("all coins supported, regtest", ...)
...
```

Reply:
Done